tracer/test/Datadog.Trace.TestHelpers/EnvironmentTools.cs

```diff
@@ -30,7 +30,7 @@ public static string GetSolutionDirectory()
             if (_solutionDirectory == null)
             {
                 var startDirectory = Environment.CurrentDirectory;
-                var currentDirectory = Directory.GetParent(startDirectory);
+                var currentDirectory = new DirectoryInfo(startDirectory); // Directory.GetParent(startDirectory);
```

Comment:
Did you mean to delete the commented out code?

Reply:
Not at first. I just didn't know why it was starting to look for the solution in the parent (locally it crashed). And yes, I must delete it now that I see it works in CI also.

Reply:
Fixed in ef976b24f589ef941d6fbb2da029a224d1597d65